scripts/sysman/memoryTimestampValidBits.yml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Memory Timestamp Valid Bits"
+version: "1.7"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Memory Timestamp Valid Bits Extension Name"
+version: "1.7"
+name: $S_MEMORY_TIMESTAMP_VALID_BITS_EXT_NAME
+value: '"$XS_extension_mem_timestamp_valid_bits"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Memory Timestamp Valid Bits Extension Version(s)"
+version: "1.7"
+name: $s_mem_ext_bandwidth_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties for Memory bandwidth"
+version: "1.7"
+class: $sMemory
+name: $s_mem_ext_bandwidth_t
```

Comment:
to have the stype, you would need a "base" field.

Now, the problem there is that that changes the sizeof of the struct, so this is breaking compatibility. the alternative would be a deprecation of s_mem_ext_bandwidth_t and named this new one with a new name.

@wdamon-intel , what do you think?

Reply:
> the problem there is that that changes the sizeof of the struct, so this is breaking compatibility. the alternative would be a deprecation of s_mem_ext_bandwidth_t and named this new one with a new name.

Correct.

Reply:
deprecated the older interface and redefined new one with updated version